### PR TITLE
chore: update vulnerability braces

### PR DESCRIPTION
**Pull request by PhilKsr**
## DESCRIPTION

Update vulnerability

### TO-DO

- [ ] ~~implement and update tests~~
- [ ] ~~update docs~~
- [x] pull `main` & resolve merge conflicts
- [x] check Vercel deployment

### CLOSES

closes https://github.com/Frachtwerk/essencium-frontend/security/dependabot/46

### CODE REVIEW

File: .nx/workspace-data/d/daemon.log
Comment (qhantom) [inline .nx/workspace-data/d/daemon.log:1]: I think we can ignore this file

**Review by qhantom — CHANGES_REQUESTED** [review]

File: .nx/workspace-data/d/daemon.log
Comment (qhantom) [reply .nx/workspace-data/d/daemon.log:1]: @PhilKsr should I have another review?

Comment (PhilKsr) [reply .nx/workspace-data/d/daemon.log:1]: Yes I removed the file and added the path to `.gitignore`. Moin!

Comment (qhantom) [reply .nx/workspace-data/d/daemon.log:1]: Yes moin!

**Review by qhantom — APPROVED** [review]
Up-Horse! THXXX

**Pull request merged**